WordPress/gutenberg
# Post Content: Display attachments

## Description

Display attachment information on the front using the post content block.
Fixes https://github.com/WordPress/gutenberg/issues/29887

## How has this been tested?




1. Activate a full site editing theme
2. Upload different types of attachments like a txt file and an image.
3. Add a caption to the image in the media library.
4. From the media library, view the attachment pages.
5. Confirm that attachments displays correctly.
6. Confirm that regular post content still display correctly.

For image attachments, the image, caption, and a link to the full size image should display.
For other formats, a link to the file should display.

 ❓  The full size image link may be too opinionated.

## Types of changes





Bug fix


## Checklist:
